### DISCUSSION TITLE
Do you have any example for video?

### DISCUSSION THREAD

**Discussion by Thanh-Binh** [halfgaar/FlashMQ#15] [General]
I am interested in sending and receiving images using MQTT.
Could you pls share me any example code?
Thanks

**Comment by halfgaar** [comment]
I don't have code ready to share. If you mean the video of piping Bad Apple as ASCII through MQTT, that is hacky code that suited my purpose, but is not ready to see daylight.

But, if you're just sending images, they are just bytes, so you can send them like any other MQTT message.

**Reply by Thanh-Binh** [reply]
thanks
